### DISCUSSION TITLE
ESP8266 support

### DISCUSSION THREAD

**Discussion by f2hamann** [khoih-prog/AsyncHTTPSRequest_Generic#7] [Q&A]
Hello,

any idea / timeline, when ESP8266 support is available. Thx!

Frank

**Comment by khoih-prog** [comment]
Sorry, no timeline yet. Possible longer time than expected as many more interesting projects are waiting in line.
Any help to add `ESP8266` support is appreciated.

**Reply by f2hamann** [reply]
Thx for your quick replay. I would like to support, but my skills need improvement to do so :-)!
I was just looking for an easy solution to access "https" (Webhooks, in my case to "Macrodroid") with ESP8266 ...

**Reply by khoih-prog** [reply]
If urgent, don't use async now. You can use regular synchronous `https` from standard ESP8266 core.

**Comment by ghost** [comment]
Would also love to see support for the ESP8266! I'am currently using the HTTP version of the library which works amazingly fast (Way faster than the synchronized HTTP/HTTPs clients...) For now, the Rest API I am calling supports unsecured HTTPS request but who knows for how much longer? I'am definitely watching 👁 this repo for new releases on the board support!

**Reply by khoih-prog** [reply]
> the HTTP version of the library which works amazingly fast (Way faster than the synchronized HTTP/HTTPs clients...)

Good that you've recognized the better Async features. 
Thanks for the request, I'll keep in mind. Hopefully that time will come sooner or later.
